examples/BUILD.bazel

```diff
@@ -17,5 +17,14 @@ npm_link_package(
     visibility = ["//examples:__subpackages__"],
 )
 
+npm_link_package(
+    name = "node_modules/@myorg/deps_lib",
+    src = "//examples/deps_lib",
+    root_package = "examples",
+    visibility = ["//examples:__subpackages__"],
+    # TODO: propagate this dependency from //examples/deps_lib: https://github.com/aspect-build/rules_ts/issues/96
+    deps = {":.aspect_rules_js/node_modules/date-fns/2.29.1": ""},
```

Comment:
nit: this might be hard for users to understand. is there an alias for this that is more readable?